lib/content-services/search/components/search-filter/search-filter.component.html

```diff
@@ -94,9 +94,9 @@
                 <mat-checkbox
                     *ngFor="let bucket of field.buckets"
                     [checked]="bucket.checked"
-                    [attr.data-automation-id]="'checkbox-'+field.label+'-'+(bucket.display || bucket.label)"
+                    [attr.data-automation-id]="'checkbox-'+(field.label | translate)+'-'+(bucket.display || (bucket.label | translate))"
```

Comment:
Needs to be i18n key, not translated value (might be multiple words with spaces)